### PR TITLE
Update pkgdown

### PR DESCRIPTION
**Pull request by Aariq**
updates _pkgdown.yml to reflect move of Alanwood database and addition of `write_mol()`.

### CODE REVIEW

**Comment by Aariq** [comment]
hmm... not sure why codecov isn't running

File: _pkgdown.yml [hunk 1]
```diff
@@ -15,9 +20,8 @@ reference:
   - etox_basic
 - title: Retrieve chemical properties
 - contents:
-  - aw_query
-  - ci_query
-  - cir_query
+  - ends_with("_query")
+  - -opsin_query
```
Comment (stitam) [inline _pkgdown.yml:24]: I'm wondering, is there a typo here?

**Review by stitam — APPROVED** [review]
Hi @Aariq thanks for the fix! I added a comment.

File: _pkgdown.yml (hunk 1 above)
Comment (Aariq) [reply _pkgdown.yml:24]: `opsin_query()` is a identifier translator, so I left it out of this section on purpose.  It shows up in the chemical identifier translator section.

**Pull request merged**